guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+
+As title, mention the symptom that needs fixing ("Cannot access...") or the error message. To quote an error message, start and end with `'` to escape `"`. for example, `title: '"FAILED" status for created ServiceInstances'`. Do not use the cause as title ("Incompatible version"), because we also want to help users who have no idea about the cause and only know something's not working as expected.
+
+It's good practice to use three standard headlines (like “Condition”, “Cause”, “Remedy”), each might have just one sentence or more as needed. For remedy, use a numbered list if there are multiple steps to follow, and a bullet list if there are several equally valid solutions.
+
+> [- DITA Troubleshooting Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-troubleshooting-topic.html)
+> In its simplest form, troubleshooting information follows this pattern:
+>
+> 1. A condition or symptom. Usually the condition or symptom is an undesirable state in a system, a product, or a service that a reader wants to correct.
+> 2. A cause for the condition or symptom.
+> 3. A remedy for the condition or symptom.
+>
+> The troubleshooting topic provides sections for describing the condition, causes, and remedies needed to restore a system, a product, or a service to normal.
+>
+> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
+>
+> Cause and remedy might occur in combinations other than pairs. It is possible to have:
+>
+> - Multiple causes with the same remedy
+> - A single cause with more than one remedy
+> - A remedy with no known cause
+> - A cause with no known remedy
+> The troubleshooting information type also can be used to document alarm clearing strategies.
+
+You can use the [troubleshooting topic template](../templates/resources/troubleshooting.md).
+
+### Release Notes
+
+Announce what's new in Kyma.
+
+After an introductory paragraph that outlines the city that's the namesake of the current release, a list briefly presents the new and changed features. Links lead to longer paragraphs that describe the changes in more detail.
+
+## Graphics
+
+### Architecture and Flow Diagrams
+
+For information about our diagram style, see [Diagrams](../02-diagrams.md).
+
+### Screenshots
+
+Wherever possible, present screenshots as simplified user interfaces (SUI). Basically, this means to blur out all UI elements that aren't essential for the task at hand.
+
+For more information, see:
+• <https://www.techsmith.com/blog/simplified-user-interface/>
+• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>
+
+## Target Groups
```

Comment:
```suggestion
## Target groups
```

You don't use capitalization in other titles. Is there a reason why `Groups` is capitalized here?

Reply:
[Old habit](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/47f574d73c8c4cc89368203af2fbafcb.html).